presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -260,13 +323,19 @@ public boolean equals(Object obj)
             }
             FunctionCall o = (FunctionCall) obj;
             return Objects.equals(getName(), o.getName()) &&
-                    Objects.equals(getArguments(), o.getArguments());
+                    Objects.equals(getArguments(), o.getArguments()) &&
+                    /*
```

Comment:
`checkState(o.getWindow.isPresent())` will throw an exception as well (similarly to `Optional.get()` on empty value) but would save you this comment.